docs/installation/appliance/docker.md

```diff
@@ -86,48 +85,38 @@ echo '{"default-address-pools":[{"base":"10.201.0.0/16","size":24}]}' | jq '.' |
 
 ## Setup your Assemblyline appliance
 
-### Download the Assemblyline docker-compose files
-
-=== "Online"
-
-    ```bash
-    mkdir ~/git
-    cd ~/git
-    git clone https://github.com/CybercentreCanada/assemblyline-docker-compose.git
-    ```
-
-### Choose your deployment type
+### Download the Assemblyline Docker Compose repository
 
-!!! important
-    After this step, we will assume that the commands that you run are from your deployment directory: ```~/deployments/assemblyline/```
-
-=== "Assemblyline only"
+```bash
+mkdir ~/git
+cd ~/git
+git clone https://github.com/CybercentreCanada/assemblyline-docker-compose.git
```

Comment:
I think we lost the `mkdir ~/deployments` and we could just clone into ~/deployments directly, since we're copying everything over.